src/test/scala/cromwell/BadDockerName.scala

```diff
@@ -0,0 +1,26 @@
+package cromwell
+
+import akka.testkit._
+import cromwell.CromwellSpec.{IntegrationTest, DockerTest}
+import cromwell.engine.WorkflowFailed
+import cromwell.util.SampleWdl
+
+import scala.language.postfixOps
+
+class BadDockerName extends CromwellTestkitSpec {
+  "A task which has a bad docker image name" should {
+    "fail properly" taggedAs DockerTest in {
+      runWdlAndAssertOutputs(
+        sampleWdl = SampleWdl.HelloWorld,
+        eventFilter = EventFilter.info(pattern = s"transitioning from Running to Failed.", occurrences = 1),
+        runtime = """
+                    |runtime {
+                    |  docker: "/fauxbuntu:nosuchversion"
+                    |}
+                  """.stripMargin,
+        expectedOutputs = Map(),
+        terminalState = WorkflowFailed
+      )
+    }
+  }
+}
```

Comment:
It was failing. On investigation, this test was in completely the wrong place. And as a result, the test it should have been did not exist.

https://github.com/broadinstitute/cromwell/commit/253530a7bfa1f5d7cbcc79fc84057072c682a1aa